target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-rutx.dtsi

```diff
@@ -0,0 +1,228 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qcom-ipq4019-ap.dk01.1.dtsi"
```

Comment:
This should probably be moved before the other includes, so the dts-v1 markup is first.

Reply:
Fixed